layouts/research/single.html

```diff
@@ -0,0 +1,17 @@
+{{ define "main" }}
+<article class="markdown">
+    <h1>{{ .Title }}</h1>
+
+    {{- if strings.ContainsNonSpace .Plain -}}
+    <p>{{ .Content }}</p>
+    {{- end -}} {{- with .Resources.ByType "page" -}} {{ range . }} {{ $section := . }}
```

Comment:
I'm fine with not having the plugin, but that is just completely unreadable.

I have this in my VS Code config:

```json
{
		"[html]": {
        "editor.defaultFormatter": "vscode.html-language-features",
        "editor.formatOnSave": false,
        "editor.formatOnPaste": false
    },
}
```